src/displayBackend/common/ConnectorBase.hpp

```diff
@@ -57,42 +57,42 @@ class ConnectorBase : public DisplayItf::Connector
 	 * @param startDirectory grant table reference to the buffer start directory
 	 * @param size           buffer size
 	 */
-	size_t getEDID(grant_ref_t startDirectory, uint32_t size);
+	size_t getEDID(grant_ref_t startDirectory, uint32_t size) const;
```

Comment:
I think, this is the good time to move all `EDID` methods and constants to a separate class/namespace, because they are providing functionality that is not strictly part of `ConnectorBase`

Reply:
agree